### PR TITLE
test/e2e/lifecycle: Update to only supported versions

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
CNAO runs lifecycle e2e test over all supported versions.
Updating versions so that the CI will test only the relevant releases.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ For more information about the configuration format check [configuring section](
 
 # Upgrades
 
-Starting with version `0.65.0`, this operator supports upgrades to any newer
+Starting with version `0.76.0`, this operator supports upgrades to any newer
```
Comment (oshoval) [inline README.md:306]: Please consider 
https://github.com/kubevirt/cluster-network-addons-operator/pull/959#discussion_r686741775

Comment (RamLavi) [reply README.md:306]: still considering :)

File: test/releases/0.85.1.go
Comment (oshoval) [inline test/releases/0.85.1.go:1]: aint those files should be added automatically ?
what command adds them ?

**Review by oshoval — APPROVED** [review]

**Pull request merged**